app/javascript/gobierto_dashboards/webapp/containers/summary/Summary.vue

```diff
@@ -116,11 +133,55 @@ export default {
       labelHalfSpendingsContracts: I18n.t('gobierto_dashboards.dashboards.contracts.summary.label_half_spendings_contracts'),
       labelContractType: I18n.t('gobierto_dashboards.dashboards.contracts.contract_type'),
       labelProcessType: I18n.t('gobierto_dashboards.dashboards.contracts.process_type'),
-      labelAmountDistribution: I18n.t('gobierto_dashboards.dashboards.contracts.amount_distribution')
+      labelAmountDistribution: I18n.t('gobierto_dashboards.dashboards.contracts.amount_distribution'),
+      labelMainAssignees: I18n.t('gobierto_dashboards.dashboards.contracts.main_assignees'),
+      labelTableThAssignee: I18n.t('gobierto_dashboards.dashboards.contracts.assignee'),
+      labelTableThContracts: I18n.t('gobierto_dashboards.dashboards.contracts.contracts'),
+      labelTableThAMount: I18n.t('gobierto_dashboards.dashboards.contracts.final_amount'),
     }
   },
+
   mounted() {
     EventBus.$emit("summary_ready");
+
+    EventBus.$on('refresh_summary_data', () => {
```

Comment:
Be careful with the event listeners throughout the code. Have this component the authority to be updated? Wouldn't be better get this payload through props? Should this event be closed on destroyed or deactivated cycles?

Reply:
Summary will not be destroyed or deactivated, so there's no need to close the event.

About using props, it might be better but I'm not sure how would I do it with the current architecture. We may discuss it on a later stage.

Reply:
Ok with the 2nd part.
But the 1st one is a good practice. Get use to close/remove/off the events once opened. The common place is in `beforeDestroy()` life hook

Reply:
Ok! I didn't know about that good practice, I'll keep that in mind. But in this case I think it does not apply, doesn't it? If Summary is never going to be destroyed.

Reply:
Do it or die.

Reply:
I'm kidding.

Reply:
Or perhaps ain't.

Reply:
So, should I do it or not or just die or what? 😅